be/src/exprs/vectorized/cast_expr.cpp

```diff
@@ -1031,6 +1031,8 @@ class VectorizedCastExpr final : public Expr {
             << ", expr=" << expr_debug_string << ")";
         return out.str();
     }
+
+    bool is_cast_expr() const override { return true; }
```

Comment:
`Expr::_opcode` could check the expression type. Use this virtual method is meaningless.